build.gradle

```diff
@@ -161,6 +164,15 @@ task apiTest(type: Test) {
   setClasspath(sourceSets.integrationTest.runtimeClasspath)
 }
 
+task functional(type: Test) {
```

Comment:
`functionalTest` wouldn't be a better name for this task. Consistent with `apiTest`

Reply:
name that is set by the CNP team, we're just implementing the interface

Reply:
That's not my decision -- that's the task CNP pipeline is expecting to call.

Reply:
ok